lang/es.json

```diff
@@ -60,14 +60,14 @@
     "Lens": "Lente",
     "New": "Nuevo",
     "Next": "Siguiente",
-    "Only Trashed": "Solo en la papelera",
+    "Only Trashed": "Sólo en la papelera",
```

Comment:
In 2010 the Royal Spanish Academy set that no accent should be used. https://spanish.stackexchange.com/questions/1123/when-does-s%C3%B3lo-have-an-accent-cu%C3%A1ndo-debe-llevar-tilde-la-palabra-solo

Reply:
Ok, I would put this into consideration, because stills valid and is more explicit to denote; "only" or "uniquely" when it's possible in spanish, and this is the case, "Sólo" explicitly says that is an adverb.

I know that RSA rule but I think that if Laravel's philosophy is to write a terse and explicit language, this could be a great detail for Spanish speakers.

Reply:
This one's my fault. I'm used to use an accent in "Solo", but it doesn't need it.

Reply:
There is still an ongoing discussion. Some popular writers say that it should have an accent, whereas others say it shouldn't. But the Royal Spanish Academy states that no accents should be used

Reply:
Ok, reading that is not adviced to use with accent, let's take off the accent and that's all!

Reply:
Sorry to intervene again... I've just asked a relative (journalist) and she thinks it'd be better with accent. My two cents there...

Reply:
I think we should base the decision on official organisms and not colleagues, our opinions, etc.